proposals/MultipleReadersWriters.md

```diff
@@ -221,6 +221,11 @@ childAccessHandle.close();
 await parentHandle.remove(); // successful
 ```
 
+### Interactions with BFCache
+The File System Access API operations/primitives hold locks in the BFCache. A page should not be made aware that it or another page is or has been in the BFCache. A fully active page could be made aware of a BFCached page if there is contention between locks they hold.
+
+To keep BFCache enabled when a site uses the File System Access API, a BFCached page must be evicted on locking contention with a fully active page. Otherwise, a file system lock held by a page will not affect the page's eligibility for BFCache. This allows the site to have the performance gains of BFCache up until it would be made aware of the BFCache.
```

Comment:
> This allows the site to have the performance gains of BFCache up until it would be made aware of the BFCache.

This sentence is a bit misleading, since it suggests there is just "the site", whereas there are two pages in question here: (1) the page in the BFCache, and (2) the active page. These pages may not correspond to the same site for local file system access